internal/service/alarms/serve.go

```diff
@@ -85,20 +89,26 @@ func Serve() error {
 
 	alarmServerStrictHandler := api.NewStrictHandlerWithOptions(&alarmServer, nil,
 		api.StrictHTTPServerOptions{
-			RequestErrorHandlerFunc:  getOranReqErrFunc(),
-			ResponseErrorHandlerFunc: getOranRespErrFunc(),
+			RequestErrorHandlerFunc:  common.GetOranReqErrFunc(),
+			ResponseErrorHandlerFunc: common.GetOranRespErrFunc(),
 		},
 	)
 
 	r := http.NewServeMux()
 
+	// This also validates the spec file
+	swagger, err := api.GetSwagger()
```

Comment:
nit: remove api from the import or use generated as in serve.go from resources. Just for consistency.

Reply:
fixed.